environments/test/test.tfvars

```diff
@@ -3,6 +3,7 @@ location               = "uksouth"
 env                    = "perftest"
 subscription           = "test"
 certificate_name_check = false
+trigger_migration      = false
```

Comment:
Why add this if the default is false?

Reply:
added in as this was going to be used for future usage, I can remove if you would like.

Reply:
removed from tfvars as been defaulted to false as highlighted